app/src/main/java/de/westnordost/streetcomplete/quests/surface/DescribeGenericSurfaceDialog.kt

```diff
@@ -0,0 +1,42 @@
+package de.westnordost.streetcomplete.quests.surface
+
+import android.content.Context
+import android.content.DialogInterface
+import android.view.LayoutInflater
+import android.widget.EditText
+import androidx.appcompat.app.AlertDialog
+import de.westnordost.streetcomplete.R
+
+class DescribeGenericSurfaceDialog(
+    context: Context,
+    onSurfaceDescribed: (txt: String) -> Unit
+) : AlertDialog(context, R.style.Theme_Bubble_Dialog) {
+    val view = LayoutInflater.from(context).inflate(R.layout.quest_surface_detailed_answer_impossible, null)
+    val explanationInput = view.findViewById<EditText>(R.id.explanationInput);
+    init {
+        setTitle(context.resources.getString(R.string.quest_surface_detailed_answer_impossible_title))
```

Comment:
I think it would make sense to explain what the user is supposed to do here. 

`R.string.quest_surface_detailed_answer_impossible_description` should be shown as a description in this dialog,

`quest_surface_detailed_answer_impossible_title` should be changed to simply _"Describe surface"_

That the error dialog that shows if the user did not provide any description simply repeats the description is fine.

Reply:
> quest_surface_detailed_answer_impossible_title should be changed to simply "Describe surface"

Done